pkg/customresourcestate/registry_factory.go

```diff
@@ -544,6 +564,7 @@ func famGen(f compiledFamily) generator.FamilyGenerator {
 	errLog := klog.V(f.ErrorLogV)
 	return generator.FamilyGenerator{
 		Name: f.Name,
+		// TODO(@rexagod): This should be dynamic.
```

Comment:
Should we create an issue for this, so it's not only a TODO sitting in the code?

Reply:
Opened https://github.com/kubernetes/kube-state-metrics/issues/1886.

Reply:
Thanks!